wao/notifier.py

```diff
@@ -20,18 +18,19 @@ def send_start_deliminator_message(brain, coin, month, year, dup, max_counter_du
     post_request(text)
 
 
-def post_request(text, is_from_429_watcher=False):
+def post_request(text, is_from_romeo=False):
     print("post_request: " + text)
-    telegram_bot_api_token = Config.NOTIFIER_TELEGRAM_BOT_API_TOKEN_429 if is_from_429_watcher else Config.NOTIFIER_TELEGRAM_BOT_API_TOKEN_BACKTEST
+    telegram_bot_api_token = Config.NOTIFIER_TELEGRAM_BOT_API_TOKEN_429 if is_from_romeo else Config.NOTIFIER_TELEGRAM_BOT_API_TOKEN_BACKTEST
     result = requests.post('https://api.telegram.org/bot' + telegram_bot_api_token +
                            '/sendMessage?chat_id=' + Config.NOTIFIER_TELEGRAM_CHANNEL_ID_BACKTEST +
-                           '&text=' + text + '&parse_mode=Markdown')
+                           '&text=' + text.replace("_", "-") + '&parse_mode=Markdown')
 
     print(str(result))
 
-    if BrainConfig.IS_429_FIX_ENABLED:
-        if str(result) != TELEGRAM_RESPONSE_200:
-            delete_429_file(text)
-            write_to_429_file(text)
-        elif str(result) == TELEGRAM_RESPONSE_200 and is_from_429_watcher:
-            delete_429_file(text)
+    if str(result) == TELEGRAM_RESPONSE_200 and is_from_romeo:
+        delete_429_file(text)
+    elif str(result) != TELEGRAM_RESPONSE_200:
```

Comment:
Two separate comments:
1. this was the initial cause of the bug - use if result is 429 
2. make this the first condition - in an if else, the first condition is the intent